rosbag2_storage/src/rosbag2_storage/metadata_io.cpp

```diff
@@ -56,14 +55,14 @@ BagMetadata MetadataIo::read_metadata(const std::string & uri)
 
 std::string MetadataIo::get_metadata_file_name(const std::string & uri)
 {
-  std::string metadata_file = (rcpputils::fs::path(uri) / metadata_filename).string();
+  std::string metadata_file = (std::filesystem::path(uri) / metadata_filename).generic_string();
```

Comment:
@MichaelOrlov I'm not 100% sure that `generic_string` is necessary. This only affects behaviour on Windows

Reply:
> @MichaelOrlov I'm not 100% sure that `generic_string` is necessary. This only affects behaviour on Windows

I believe that's why we added it earlier, but we'll see what Windows CI has to say about this code.